docs/develop/build-a-dApp/build-smart-contracts/evm_extensions.md

```diff
@@ -8,17 +8,20 @@ sidebar_position: 3
-|----------------------------------------------|-----------------|----------|---------------------------------------------------|
-| `0x0000000000000000000000000000000000000001` | ecRecover       | No       |                                                   |
-| `0x0000000000000000000000000000000000000002` | SHA256 Hash     | No       |                                                   |
-| `0x0000000000000000000000000000000000000003` | RIPEMD-160 Hash | No       |                                                   |
-| `0x0000000000000000000000000000000000000004` | Data Copy       | No       |                                                   |
-| `0x0000000000000000000000000000000000000005` | ExpMod          | No       | [EIP-198](https://eips.ethereum.org/EIPS/eip-198) |
-| `0x0000000000000000000000000000000000000006` | BN256Add        | No       | [EIP-196](https://eips.ethereum.org/EIPS/eip-196) |
-| `0x0000000000000000000000000000000000000007` | BN256ScalarMul  | No       | [EIP-196](https://eips.ethereum.org/EIPS/eip-196) |
-| `0x0000000000000000000000000000000000000008` | BN256Pairing    | No       | [EIP-197](https://eips.ethereum.org/EIPS/eip-197) |
-| `0x0000000000000000000000000000000000000009` | Blake2F         | No       | [EIP-152](https://eips.ethereum.org/EIPS/eip-152) |
+| Address                                      | Name                | Stateful | EIP                                               |
+| -------------------------------------------- | ------------------- | -------- | ------------------------------------------------- |
+| `0x0000000000000000000000000000000000000001` | ecRecover           | No       |                                                   |
+| `0x0000000000000000000000000000000000000002` | SHA256 Hash         | No       |                                                   |
+| `0x0000000000000000000000000000000000000003` | RIPEMD-160 Hash     | No       |                                                   |
+| `0x0000000000000000000000000000000000000004` | Data Copy           | No       |                                                   |
+| `0x0000000000000000000000000000000000000005` | ExpMod              | No       | [EIP-198](https://eips.ethereum.org/EIPS/eip-198) |
+| `0x0000000000000000000000000000000000000006` | BN256Add            | No       | [EIP-196](https://eips.ethereum.org/EIPS/eip-196) |
+| `0x0000000000000000000000000000000000000007` | BN256ScalarMul      | No       | [EIP-196](https://eips.ethereum.org/EIPS/eip-196) |
+| `0x0000000000000000000000000000000000000008` | BN256Pairing        | No       | [EIP-197](https://eips.ethereum.org/EIPS/eip-197) |
+| `0x0000000000000000000000000000000000000009` | Blake2F             | No       | [EIP-152](https://eips.ethereum.org/EIPS/eip-152) |
+| `0x0000000000000000000000000000000000000010` | Bech32 encoding     | No       |                                                   |
```

Comment:
actually, this address is wrong. See the Notion doc with the guidelines

Reply:
@fedekunze is like that on the extensions repo. Could you please indicate which is the notion doc that you're referring to?